bin/ingest-gisaid

```diff
@@ -37,15 +37,29 @@ esac
 echo "S3_SRC is $S3_SRC"
 echo "S3_DST is $S3_DST"
 
+fetch=0
+
+for arg; do
+    case "$arg" in
+        --fetch)
+            fetch=1;;
+        *)
+    esac
+done
+
```

Comment:
Taking a look at this PR now, but I have a small request: now that we have arguments, could we add a `--help` function?

Reply:
e.g. [this one in Seattleflu codebase](https://github.com/seattleflu/backoffice/blob/85be1faa2835ef7fc5b2848cd914b7bd4c229baa/bin/generate-and-upload-uw-retro-redcap-dets#L38)